packages/clerk-js/src/core/resources/User.ts

```diff
@@ -250,6 +252,10 @@ export class User extends BaseResource implements UserResource {
     });
   };
 
+  getOrganizationInvitations = (params?: GetUserOrganizationInvitations) => {
```

Comment:
I think we should either describe the whole method with a type (`getOrganizationInvitations` -> `type GetOrganizationInvitations`) or just the params (`params` -> `type GetUserOrganizationInvitationsParams`)

If unsure, let's start by typing the params. We can reuse the type if needed to create `GetOrganizationInvitations` in the future. 

```suggestion
  getOrganizationInvitations = (params?: GetUserOrganizationInvitationsParams) => {
```